src/cisco_gnmi/client.py

```diff
@@ -277,6 +277,7 @@ def subscribe_xpaths(
         sample_interval=_NS_IN_S * 10,
         suppress_redundant=False,
         heartbeat_interval=None,
+        prefix=None
```

Comment:
Please add the expected type in doc here - a Path? String?

Reply:
Added docstring.